src/main/java/com/solvd/blog/neo4j/NeoUsers.java

```diff
@@ -17,13 +18,14 @@
 public class NeoUsers implements Users {
 
     private final Driver driver;
-    private final UserMapper mapper;
+    private final UserMapper userMapper;
+    private final BooleanMapper booleanMapper;
 
     @Override
     public List<User> iterate() {
         try (Session session = driver.session()) {
             return session.run("MATCH (user:User) RETURN user")
-                    .list(this.mapper::toEntity);
+                    .list(this.userMapper::toEntity);
```

Comment:
@hizmailovich what is the purpose of these changes?

Reply:
@h1alexbel because I added BooleanMapper